README.md

```diff
@@ -4,3 +4,7 @@ This repo is under construction and no data models have been released.
 ### List of data models
```

Comment:
This file is not necessary. It is generated automatically. See --don't do these-- section 4 in [contribution manual](https://bit.ly/contribution_manual)

Reply:
Deleted README.md